doc/nrf/releases/release-notes-latest.rst

```diff
@@ -144,6 +149,11 @@ sdk-nrfxlib
 
 See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for the most current information.
 
+Multiprotocol Service Layer (MPSL)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Modified the nRF5340 to use SWI by default, instead of EGU.
```

Comment:
What have we modified here? Surely not the nRF5340 SoC, which is what we're saying at the moment. ;)